chart/values.yaml

```diff
@@ -442,7 +444,7 @@ webserver:
   #   CSRF_ENABLED = True
 
   service:
-    type: ClusterIP
+    type: NodePort
```

Comment:
To use Ingress, this service needs to be exposed as a NodePort to have an external IP.